manage/installing/installation.rst

```diff
@@ -125,11 +125,11 @@ Download the latest Plone unified installer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Download from `the plone.org download page <https://plone.org/download>`_ to your server using wget command. Curl also works.
```

Comment:
FYI: https://plone.org/download redirects to 5.0.2 and its not easy to get to the 4.3.8 release from there. So we may want to point directly to https://plone.org/products/plone/releases/4.3.8